Modules/User/Services/UserGetService.php

```diff
@@ -40,18 +40,24 @@ public function info($id) : ?User {
     }
 
     /**
-     * @param $id
+     * @param array $data
      * @return User|null
      * At this point everything is validated, we shouldn't check anything else
      * TODO this is just a test  function to show backend and frontend connection
      */
-    public function list() : ?Collection {
-        $companies= $this->userRepo->all();
-
+    public function list($data) : ?Collection {
+        $users = array();
+        foreach($data AS $clave=>$value){            
```

Comment:
mmm para que hiciste esto aqui?
creo no necesitas el foreach
puedes hacer
$this->userRepo->whereIn($ids); //$ids es un arreglo de ids $ids = [1, 2, 3, 4]